plugins/govppmux/plugin_impl_govppmux.go

```diff
@@ -178,6 +212,51 @@ func (p *Plugin) AfterInit() error {
 	p.wg.Add(1)
 	go p.handleVPPConnectionEvents(ctx)
 
+	if p.config.ProxyEnabled {
+		registerMsgs := func(msgs []govppapi.Message) {
+			for _, msg := range msgs {
+				gob.Register(msg)
+			}
+		}
+
+		apiChan, err := p.vppConn.NewAPIChannel()
+		if err != nil {
+			return err
+		}
+		defer apiChan.Close()
+
+		// register gob
+		for ver, h := range vppcalls.Versions {
+			if err = apiChan.CheckCompatiblity(h.Msgs...); err != nil {
+				continue
+			}
+			registerMsgs(h.Msgs)
+			registerMsgs(ifplugin.Versions["vpp"+strings.Replace(ver, ".", "", - 1)].Msgs)
```

Comment:
there should be no space between "-" and "1" :cop:
(most editors supports (via some plugin) running "gofmt" on file save action)

Also, how did it happen that _"vppXXYY"_ for all packages inside vpp plugin and _"XX.YY"_ for telemetry and govppmux? :)

Reply:
It's because it has not been refactored yet since it was introduced. I'm actually just working on this.